### ISSUE TITLE
Inform the user when the `passwords` collection is not being synced.

### ISSUE THREAD

**Issue by nickbrandt** [mozilla-lockwise/lockwise-android#186]
Context: the user can chose which collections they want synced. If the `passwords` collection is not being synced, then Lockbox won't work.

From @nickalexander

> Lockbox should message when the "passwords" Sync 1.5 collection is not being synced.  That is, if a user's Sync 1.5 account is healthy, but the meta/global record doesn't enable "passwords", then Lockbox should message as much.  (In this case Lockbox actually could remotely enable syncing passwords, but I think that's a lot of risk that you probably don't want to sign up to test.)
>
> To test this situation, connect Lockbox to an existing Sync 1.5 account.  Have a different client (they all support this behaviour) disable syncing "passwords" (called "Logins" everywhere, I think).  You'll probably find that Lockbox thinks everything is fine -- and from Sync 1.5's perspective, it is.  But you'll suddenly have no data locally, since the Sync 1.5 code you're using recognizes this situation and wipes the local collection.  (Sync 1.5's model is that there's one setting for "am I syncing this datatype" for the whole device constellation; it's not per-device.)
>
> Concretely, we need to ensure that _something_ checks that "passwords" is enabled in meta/global and exposes that to Lockbox.  One way to achieve that might be to expand `SyncError` to include `PasswordsCollectionDisabled`, and then to use the `reasonToNotSync` function to bubble up the error to Lockbox.  I don't see a low-impact way to do this, since by necessity it spans the two active code bases.

### Visual Design
**InVision**
No Entries (during initial sync): https://mozilla.invisionapp.com/share/3SN35UHWJDX#/329017812_Logins_Not_Set_To_Sync_-_No_Entries
Has Entries (after initial sync): https://mozilla.invisionapp.com/share/3SN35UHWJDX#/329017813_Logins_Not_Set_To_Sync_-_With_Entries

**Zeplin**
No Entries (during initial sync): https://zpl.io/VD4oqYw
Has Entries (after initial sync): https://zpl.io/25woG3o

Note: The "Show me how" and "Learn more" links would direct to the FAQ, specifically the "How do I enable sync on Firefox?" section. (https://lockbox.firefox.com/faq.html#how-do-i-enable-sync-on-firefox)

**Comment by devinreams** [comment]
@linuxwolf @ncalexan can you help us more concretely understand the probability of this happening to a user (low but in a realm of possibility? and what the impact would be (no work-around possible? they're stuck with no data so HIGH?)

**Comment by devinreams** [comment]
Context and detail was shared on Slack in order to help product make a prioritization decision:
https://mozilla.slack.com/archives/C4LNYNJ8G/p1550097920027700

tldr: "since I can't remember it being a severe issue with our iOS launch, I'm supposing it won't be a severe issue for Android, and therefore we don't have to make this happen for MVP?" which seemed sound 👍

**Comment by ncalexan** [comment]
> @linuxwolf @ncalexan can you help us more concretely understand the probability of this happening to a user (low but in a realm of possibility? and what the impact would be (no work-around possible? they're stuck with no data so HIGH?)

I think you've already made a decisison, but let me weigh in.  The probability of this occurring is low but not negligible.  I expect Leif (better yet, Bob) could tell you how many Sync 1.5 accounts there are that have been active within the last N days and have the "passwords" collection disabled.

The impact isn't all that high, in my opinion.  You will have a confused user: such a user will quickly find out they can't address the situation in Lockbox, and indeed they can't even test in Lockbox 'cuz they can't create a login.  So they'll go to a real browser, and try to save a login.  It'll work, 'cuz the logins feature in Desktop and the Sync 1.5 setting are not integrated together.  They'll force a sync (if they're savvy) or just wait.  They'll return to Lockbox and it'll never show up.  That user will, I expect, be lost: either at this point or more likely well before it.  But that's not the same as a "lost data" user.

**Comment by linuxwolf** [comment]
@ncalexan Thank you for weighing in.  Your own assessment (plus the metrics feedback that Leif provided) reaffirm our stance that this is nice to have but not critical for MVP.

There is a FAQ entry (that gets linked to in-app) that nudges someone to check a few things, including that "sync logins" are checked.  It requires that someone to go read the FAQ, but we've provided some breadcrumbs.

**Comment by eliserichards** [comment]
No longer valid - timeout has been removed and deprioritized (see #889).
